pkg/apply/common_test.go

```diff
@@ -64,10 +64,14 @@ func (i inventoryInfo) toUnstructured() *unstructured.Unstructured {
 	}
 }
 
-func (i inventoryInfo) toWrapped() inventory.Info {
+func (i inventoryInfo) toInfo() inventory.Info {
```

Comment:
Why do we have both `inventoryInfo` and `inventory.Info`?

Reply:
Ah, struct & interface.

Do we still need this to be an interface or could we just use a public struct?